website/src/css/custom.less

```diff
@@ -0,0 +1,521 @@
+}
+
+.footnote:hover {
+    color: #000 !important;
+}
+
+header>h1 {
+    font-size: 2rem !important;
+    margin-top: 20px !important;
+}
+
+.markdown>h2 {
+    font-size: 1.5rem !important;
+}
+
+.markdown>h3 {
+    font-size: 1.2rem !important;
+}
+
+.markdown>img {
+    display: block;
+    margin: auto;
+    margin-top: 15px;
+    margin-bottom: 20px;
+    border-radius: 6px;
+    overflow: hidden;
+    width: auto;
+    max-width: 100%;
+}
+
+.panel {
+    border-top: 2px solid #ddd;
+    padding: 20px 20px 25px 20px;
+}
+
+html[data-theme='dark'] .panel {
+    border-top: 2px solid #555;
+}
+
+.panel>a,
+.panel>h2 {
+    text-transform: uppercase;
+    font-size: 20px;
+    letter-spacing: 0.7px;
+    font-weight: 500;
+    margin-bottom: 0;
+    line-height: 33px;
+}
+
+.panel>a,
+.panel>h2 {
+    display: inline-block;
+}
+
+.panel>h3 {
+    display: block;
+    text-transform: uppercase;
+    font-size: 17px;
+    letter-spacing: 0.6px;
+    font-weight: 500;
+    margin-bottom: 0;
+    line-height: 28px;
+}
+
+.panel>p {
+    font-size: 15px;
+    margin-bottom: 0;
+}
+
+.panel.section-title {
+    background-color: #f9f9f9;
+    border-left: 2px solid #ddd;
+    border-right: 2px solid #ddd;
+}
+
+html[data-theme='dark'] .panel.section-title {
+    background-color: #292929;
+    border-top: 2px solid #555;
+    border-left: 2px solid #555;
+    border-right: 2px solid #555;
+}
+
+.panel.secondary-panel.section-title {
+    background-color: #fefefe;
+    border-left: 2px solid #ddd;
+    border-right: 2px solid #ddd;
+}
+
+html[data-theme='dark'] .panel.secondary-panel.section-title {
+    background-color: #2e2e2e;
+    border-top: 2px solid #555;
+    border-left: 2px solid #555;
+    border-right: 2px solid #555;
+}
+
+.panel.secondary-panel {
+    padding: 15px 20px 20px 20px;
+}
+
+.update-request {
+    margin-left: auto;
+    display: block;
+    text-align: right;
+    margin-top: 10px;
+}
+
+
+
+
+
+/* see: https://github.com/facebook/docusaurus/pull/7134/files  */
+/* referenced by: https://github.com/facebook/docusaurus/pull/7134 */
+/* as a fix for wrapping sidebar titles causing buggy behavior (in our case, docs were getting cut off in nested sidebar expandos because after expanding the nested section, a scrollbar appears, which pushes the sidebar content over, but only after docusaurus reads the inner element height, causing the inner sidebar section to expand its height a little less than it needs to, causing bottom sidebar entries to appear cut off. */
+.menu {
+    padding: 0.5rem 0 0.5rem 0.5rem;
+    scrollbar-gutter: stable;
+}
+
+.lastUpdated_3DPF {
+    display: none;
+}
+
+nav.pagination-nav {
+    margin-top: 0;
+    margin-bottom: 20px;
+}
+
+article {
+    margin-bottom: 10px !important;
+}
+
+.footer {
+    padding: 10px !important;
+    display: none !important;
+    /* can remove display style if we want to reintroduce */
+}
+
+footer>.col {
+    text-align: right;
+}
+
+footer>.col a,
```

Comment:
these can all be grouped and organized using LESS conventions in a separate PR - this CSS is old CSS pulled into the new LESS file